app/views/equipments/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1 class='repair-header d-flex a-center j-center'>New Equipment</h1>
+<div class='add-equipment d-flex f-col a-center j-center'> 
+    <%=form_with url: equipments_path, scope: :data do |f| %>
+      <div class="field3 d-flex a-center j-center f-col" >
+        <%= f.label :icon, 'Select Icon' %>
+         <%= f.select :icon, @icons.collect { |icon| [icon['name'], icon['image']] }, class: "full-width", :prompt => 'Select an Icon', :required=>true %>
+      </div>
+      <div class="field3 d-flex a-center j-center f-col">
+        <%= f.label :name, 'Name'  %>
+        <%= f.text_field :name, :required=>true %>
+      </div>
+    <div class="action1 d-flex a-center f-col">
+         <%= f.submit "Add Equipment"%>
+    </div>
+      
+<% end %>
+</div>
```

Comment:
- [ ] When the user clicks on the "Back" button (<), the user navigates to the home page, please implement the back button when it comes to the mobile version.

![image](https://user-images.githubusercontent.com/55635977/158816849-847ae3f8-ee6a-4df4-a569-6ed24675457e.png)